src/index.js

```diff
@@ -8,8 +8,15 @@ const cache = new FileCache();
 const loader = new Loader(cache);
 
 const vpWebBuildTime = timeago.format(global.VPWEB_BUILD_TIMESTAMP);
+const vpxJsBuildTime = timeago.format(global.VPXJS_BUILD_TIMESTAMP);
+const detailText = 'VPX-JS build ' + vpxJsBuildTime +
+	'\nVPX-JS GIT HASH ' + global.VPXJS_GIT_HASH +
+	'\nVPX-JS GIT BRANCH ' + global.VPXJS_GIT_BRANCH +
+	'\nVPWEB v' + global.VPWEB_VERSION + ', built ' + vpWebBuildTime;

```

Comment:
So now you have:
> VPX-JS build 8 minutes ago
> VPX-JS GIT HASH d8c0641
> VPX-JS GIT BRANCH feature/build-information-second-try
> VPWEB v1.0.0-a6b364f, built 42 seconds ago

A lot of lines. How about:

> VPX-JS feature/build-information-second-try@d8c0641 (8 minutes ago)
> VPWeb v1.0.0-a6b364f (42 seconds ago)

Reply:
>Looking good! I'm just surprised that global is available in the browser.

its global you know ;)


i like the compact view!